src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-
+spring.data.mongodb.uri=mongodb://admin:admin1@ds149489.mlab.com:49489/heroku_dmv0bt1f?retryWrites=false
```

Comment:
Use environment variables for this url. Heroku provides mongoDB environment variable and you can create one for your own pcs.

Reply:
Done